frontend/src/styles.scss

```diff
@@ -93,4 +93,22 @@ form[data-amplify-authenticator-setup-totp] {
   div[data-amplify-copy] {
     display: none;
   }
+
+}
+
+.banner_header {
+  padding-left: 195px;
+  padding-top: 10px;
+  transition: top .2s;
+  background-color: #DDEFF5;
+  font-weight: bold;
+}
+
+.banner_login {
```

Comment:
I'd suggest we consider using a uswds component instead (such as https://designsystem.digital.gov/components/card/ or https://designsystem.digital.gov/components/summary-box/) -- since we're not basing this off of a Figma design at the moment, when we add new content to the site, it's probably best to stick with the existing components so they "fit in" with the theme as opposed to making our styles.